src/ddmd/root/port.d

```diff
@@ -38,7 +38,7 @@ extern (C++) struct Port
     {
         int result = 0;
 
-        for (int i = 0; i < n; i++)
+        for (size_t i = 0; i < n; i++)
```

Comment:
Might as well go whole hog with:
```
foreach (i; 0 .. n)
```

Reply:
Right, I'm still stuck in doing C++-isms.